modules/apps/knowledge-base/knowledge-base-editor-configuration/src/main/java/com/liferay/knowledge/base/editor/configuration/internal/KBContentCKEditorConfigContributor.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.knowledge.base.editor.configuration.internal;
+
+import com.liferay.knowledge.base.constants.KBPortletKeys;
+import com.liferay.portal.kernel.editor.configuration.BaseEditorConfigContributor;
+import com.liferay.portal.kernel.editor.configuration.EditorConfigContributor;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.portlet.RequestBackedPortletURLFactory;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Ambrín Chaudhary
+ */
+@Component(
+	property = {
+		"editor.config.key=contentEditor", "editor.name=ckeditor",
+		"javax.portlet.name=" + KBPortletKeys.KNOWLEDGE_BASE_ADMIN,
+		"javax.portlet.name=" + KBPortletKeys.KNOWLEDGE_BASE_ARTICLE,
+		"javax.portlet.name=" + KBPortletKeys.KNOWLEDGE_BASE_DISPLAY,
+		"javax.portlet.name=" + KBPortletKeys.KNOWLEDGE_BASE_SEARCH,
+		"javax.portlet.name=" + KBPortletKeys.KNOWLEDGE_BASE_SECTION
+	},
+	service = EditorConfigContributor.class
+)
+public class KBContentCKEditorConfigContributor
+	extends BaseEditorConfigContributor {
+
+	@Override
+	public void populateConfigJSONObject(
+		JSONObject jsonObject, Map<String, Object> inputEditorTaglibAttributes,
+		ThemeDisplay themeDisplay,
+		RequestBackedPortletURLFactory requestBackedPortletURLFactory) {
+
+		jsonObject.put(
+			"toolbar", "kb"
+		).put(
+			"toolbar_kb", getToolbarKBJSONArray(inputEditorTaglibAttributes)
+		);
+	}
+
+	protected JSONArray getToolbarKBJSONArray(
+		Map<String, Object> inputEditorTaglibAttributes) {
+
+		JSONArray jsonArray = JSONUtil.putAll(
+			super.toJSONArray("['Bold', 'Italic', 'Underline']"),
+			super.toJSONArray("['NumberedList', 'BulletedList']"),
+			super.toJSONArray("['Styles']"),
+			super.toJSONArray("['Link', 'Unlink']"),
+			super.toJSONArray("['Table','ImageSelector','HorizontalRule']"));
+
+		//TODO see CKEditorConfigContributor L:205
```

Comment:
@adolfopa  please see this TODO. I don't know if it is possible to access that method or we should duplicate it

Reply:
IMO it's not worth it; we don't need all the additional logic that the base class brings with it, so duplicating that method is the lesser evil ;-)

Reply:
Let's keep things simple; move that logic, but as a private method instead of a protected one and call it a day.